rest-api-tests.http

```diff
@@ -0,0 +1,154 @@
+// run the server first
```

Comment:
what the heck is that file

Reply:
Some info on how to run that would be useful